ocsmesh/__init__.py

```diff
@@ -4,7 +4,25 @@
 import os
 import sys
 import platform
+import warnings
 
+# Major API Change Warning
+warnings.warn(
+    "\n\n"
+    "********************************************************************************\n"
+    "                  MAJOR UPDATE ALERT - Upcoming OCSMesh 2.0.0                   \n"
+    "********************************************************************************\n"
+    "OCSMesh is going through significant architectural changes:\n"
+    "1. Mesh Engines (Jigsaw and Triangle) will become OPTIONAL dependencies.\n"
+    "   You must install them manually if needed.\n"
+    "2. A new default mesh engine will be adopted to replace Jigsaw.\n"
+    "3. 'jigsaw_msh_t' has been replaced by 'ocsmesh.internal.MeshData'.\n"
+    "4. Functions operating on msh_t objects are deprecated and will be updated to support meshdata.\n"
+    "Please check the release notes and update your workflows accordingly.\n"
```

Comment:
Are you going to add more details about upcoming v2 in release notes? Or are you referring to v2 release notes when it's released?